business-registry-dashboard/app/components/Table/AffiliatedEntity/Action.vue

```diff
@@ -251,10 +251,10 @@ const handleApprovedNameRequestRenew = async (item: Business): Promise<void> =>
   const corpNum = item.nameRequest?.corpNum
   if (!corpNum) { return } // Early return if no corpNum exists
 
-  // If user is not staff, redirect to steps to restore page
-  if (!accountStore.isStaffOrSbcStaff) {
+  // redirect to steps to restore page if authorized (non-staff)
+  if (IsAuthorized(AuthorizedActions.REDIRECT_STEP_TO_RESTORE_PAGE)) {
```

Comment:
I think it would be better to make this staff-centric. In other words, keep this as a "not" condition. (Same with line 140, actually.)

Secondly, isn't this the same condition as line 140? If the flag is one way then the tooltip tells you about the forms and this code navigates you there? I think line 165 is also basically the same condition. Thoughts?